backend/src/db/migrations/meta/0000_snapshot.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
what is that ?

Reply:
It's needed by drizzle to generate migrations.